package.json

```diff
@@ -51,10 +51,10 @@
   },
   "license": "MIT",
   "devDependencies": {
-    "@typescript-eslint/eslint-plugin": "^6.11.0",
-    "@typescript-eslint/parser": "^6.11.0",
-    "bun-types": "latest",
-    "eslint": "^8.53.0",
+    "@typescript-eslint/eslint-plugin": "^6.16.0",
+    "@typescript-eslint/parser": "^6.16.0",
+    "bun-types": "^1.0.20",
+    "eslint": "^8.55.0",
     "eslint-plugin-import": "^2.29.1"
```

Comment:
why we are changing dependencies is there any specific reason

Reply:
Just staying up to date.

Reply:
i guess `latest` can be stay for `bun-types`